src/bin/bat/app.rs

```diff
@@ -238,6 +238,10 @@ impl App {
                 .map(LineRanges::from)
                 .map(|lr| HighlightedLineRanges(lr))
                 .unwrap_or_default(),
+            offset: match self.matches.value_of("offset") {
+                Some(v) => v.parse::<usize>().unwrap_or(0),
+                _ => 0,
+            }
```

Comment:
You might be able to use `map_or` here:
```
offset: self.matches.value_of("offset").map_or(…)
```

Reply:
Is something like d3e2bb3346ceaa0c4a3fcb6aab19920f1667f1c1 what you had in mind?